diff -urN imlib2-1.1.0.orig/configure.ac imlib2-1.1.0/configure.ac
--- imlib2-1.1.0.orig/configure.ac	Thu Aug 30 04:40:08 2001
+++ imlib2-1.1.0/configure.ac	Mon Jan 21 12:02:43 2002
@@ -43,7 +43,9 @@
 
 AC_C_BIGENDIAN	      
 
-AC_LIBLTDL_CONVENIENCE
+AC_LIBLTDL_INSTALLABLE
+AC_SUBST(LTDLINC)
+AC_SUBST(LIBLTDL)
 AC_CONFIG_SUBDIRS(libltdl)
 
 AC_PROG_CC
--- imlib2-1.1.0/src/Makefile.am.orig	2003-09-08 04:12:09.000000000 +0200
+++ imlib2-1.1.0/src/Makefile.am	2003-10-06 21:37:53.749441656 +0200
@@ -7,7 +7,7 @@
 
 LDFLAGS		     = -L/usr/X11R6/lib
 INCLUDES             = -I. -I$(top_srcdir) -I$(top_srcdir)/src \
-                       -I$(top_srcdir)/loaders -I$(top_srcdir)/libltdl \
+                       -I$(top_srcdir)/loaders $(LTDLINCL) \
                        $(freetype_cflags) -I/usr/X11R6/include $(X_CFLAGS) \
                        -I$(prefix)/include -I$(includedir)
 MMX_SRCS             = asm_blend.S asm_rgba.S asm_scale.S asm_rotate.S \
@@ -32,12 +32,12 @@
 
 if HAVE_MMX
 libImlib2_la_LIBADD   = $(X_OBJS) @DLLDFLAGS@ $(MMX_OBJS) \
-                       $(top_builddir)/libltdl/libltdlc.la \
+                       $(LIBLTDL) \
                        $(x_libs) $(freetype_libs) $(LDFLAGS)
 libImlib2_la_DEPENDENCIES = $(top_builddir)/config.h $(X_OBJS) $(MMX_OBJS)
 else
 libImlib2_la_LIBADD   = $(X_OBJS) @DLLDFLAGS@ \
-                       $(top_builddir)/libltdl/libltdlc.la \
+                       $(LIBLTDL) \
                        $(x_libs) $(freetype_libs) $(LDFLAGS)
 libImlib2_la_DEPENDENCIES = $(top_builddir)/config.h $(X_OBJS)
 endif
