--- imlib2-1.0.4/configure.in.orig	Mon Nov 12 22:41:31 2001
+++ imlib2-1.0.4/configure.in	Mon Nov 12 22:42:16 2001
@@ -44,7 +44,6 @@
 AC_C_BIGENDIAN	      
 
 AC_LIBLTDL_CONVENIENCE
-AC_CONFIG_SUBDIRS(libltdl)
 
 AC_PROG_CC
 AM_PROG_CC_STDC
@@ -158,7 +157,7 @@
 fi
 fi
 fi
-AC_CHECK_LIB(dl, dlopen, dlopen_libs=-ldl)
+AC_CHECK_LIB(dl, dlopen, dlopen_libs=-ldl, [], -ldl)
 
 EXTRA_LIBS="-L/usr/local/lib -L/usr/X11R6/lib"
 
