diff -urN imlib2-1.0.4/configure.in imlib2-1.0.4.new/configure.in
--- imlib2-1.0.4/configure.in	Mon Dec  3 01:10:36 2001
+++ imlib2-1.0.4.new/configure.in	Mon Dec  3 01:09:52 2001
@@ -6,6 +6,8 @@
         [CPPFLAGS="$CPPFLAGS -I$withval/include"
         LIBS="-L$withval/lib $LIBS"])
 
+AM_PROG_AS
+
 mmx=no
 AC_ARG_ENABLE(mmx,[  --enable-mmx            attempt compiling using mmx assembly [default=yes]],
 [
--- imlib2-1.0.6/src/Makefile.am.wiget	Sat Aug 10 14:55:49 2002
+++ imlib2-1.0.6/src/Makefile.am	Sat Aug 10 14:56:43 2002
@@ -10,6 +10,7 @@
                        $(X_CFLAGS) -I$(prefix)/include -I$(includedir) \
 		       -I. -I$(top_srcdir) -I$(top_srcdir)/src \
 		       -I$(top_srcdir)/loaders
+AM_CCASFLAGS         = $(INCLUDES)		       
 MMX_SRCS             = asm_blend.S asm_rgba.S asm_scale.S asm_rotate.S \
 		       asm_blend_cmod.S
 MMX_OBJS             = $(MMX_SRCS:.S=.lo)
