--- imlib2-1.1.1/configure.in.orig	2004-07-21 02:42:34.000000000 +0200
+++ imlib2-1.1.1/configure.in	2004-08-01 10:59:59.003630864 +0200
@@ -17,34 +17,6 @@
 AM_ENABLE_SHARED
 AM_PROG_LIBTOOL
 
-if test "x${exec_prefix}" = "xNONE"; then
-  if test "x${prefix}" = "xNONE"; then
-    bindir="${ac_default_prefix}/bin";
-  else
-    bindir="${prefix}/bin";
-  fi
-else
-  if test "x${prefix}" = "xNONE"; then
-    bindir="${ac_default_prefix}/bin";
-  else
-    bindir="${prefix}/bin";
-  fi
-fi
-
-if test "x${exec_prefix}" = "xNONE"; then
-  if test "x${prefix}" = "xNONE"; then
-    libdir="${ac_default_prefix}/lib";
-  else
-    libdir="${prefix}/lib";
-  fi
-else
-  if test "x${prefix}" = "xNONE"; then
-    libdir="${ac_default_prefix}/lib";
-  else
-    libdir="${prefix}/lib";
-  fi
-fi
-
 dnl Set PACKAGE_BIN_DIR in config.h.
 if test "x${bindir}" = 'xNONE'; then
   if test "x${prefix}" = "xNONE"; then
