--- imlib2-1.1.0/src/Makefile.am.orig	2003-10-13 23:07:43.000000000 +0200
+++ imlib2-1.1.0/src/Makefile.am	2003-10-13 23:09:03.200487184 +0200
@@ -33,12 +33,12 @@
 if HAVE_MMX
 libImlib2_la_LIBADD   = $(X_OBJS) @DLLDFLAGS@ $(MMX_OBJS) \
                        $(LIBLTDL) \
-                       $(x_libs) $(freetype_libs) $(LDFLAGS)
+                       $(x_libs) $(freetype_libs) $(LDFLAGS) -lm
 libImlib2_la_DEPENDENCIES = $(top_builddir)/config.h $(X_OBJS) $(MMX_OBJS)
 else
 libImlib2_la_LIBADD   = $(X_OBJS) @DLLDFLAGS@ \
                        $(LIBLTDL) \
-                       $(x_libs) $(freetype_libs) $(LDFLAGS)
+                       $(x_libs) $(freetype_libs) $(LDFLAGS) -lm
 libImlib2_la_DEPENDENCIES = $(top_builddir)/config.h $(X_OBJS)
 endif
 
